package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "tslint -c tslint.json -e \"**/node_modules/**\" -p ./tsconfig.json",
     "lint:fix": "tslint -c tslint.json --fix -e \"**/node_modules/**\" -p ./tsconfig.json",
     "lint-staged": "lint-staged",
-    "postbuild": "npm prune --production",
+    "clean-pre-prod": "npm prune --production",
```

Comment:
why do we need this? postbuild runs automatically

Reply:
I found it in the `toolPipeline`, and thought it's a good idea. It makes it a bit easier to work with the repo, so you can build it locally without deleting `devDelendencies`.

Reply:
alright :)